app/templates/add_buyer_email_domain.html

```diff
@@ -0,0 +1,82 @@
+{% extends "_base_page.html" %}
+{% block page_title %}
+  Add buyer email domains – Digital Marketplace admin
+{% endblock %}
+
+{% block breadcrumb %}
+  {%
+    with items = [
+      {
+        "link": url_for('.index'),
+        "label": "Admin home"
+      },
+      {
+        "label": "Add buyer email domains"
+      }
+    ]
+  %}
+    {% include "toolkit/breadcrumb.html" %}
+  {% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  {% with messages = get_flashed_messages(with_categories=true) %}
+  {% if messages %}
+      {% for category, message in messages %}
+          {%
+          with
+            message = message,
+            type = "destructive" if category == 'error' else 'success'
+          %}
+          {% include "toolkit/notification-banner.html" %}
+          {% endwith %}
+      {% endfor %}
+  {% endif %}
+  {% endwith %}
+
+  {% for error in email_domain_form.new_buyer_domain.errors %}
+    {%
+      with
+        message = error,
+        type = "destructive"
+    %}
+      {% include "toolkit/notification-banner.html" %}
+    {% endwith %}
+  {% endfor %}
+
+  {% with heading = "Add a buyer email domain" %}
+    {% include "toolkit/page-heading.html" %}
+  {% endwith %}
+
+  <form method="post">
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <input id="csrf_token" name="csrf_token" type="hidden" value="{{ csrf_token() }}">
+        <p>
+          <a href="https://www.gov.uk/government/publications/public-sector-organisations-eligible-to-use-cloudstore">
+            Check 'Public sector organisations eligible to use Digital Marketplace'
+          </a> before you add a domain.
+        </p>
+
+        {%
+          with
+          question = "",
+          hint = "For example police.uk",
+          name = email_domain_form.new_buyer_domain.name,
+          value = email_domain_form.new_buyer_domain.data,
+          error = ""
```

Comment:
I think `error` is optional for the textbox template so you might be able to leave it out (I think `question` is required though).

Reply:
Yes, I was leaving it in to explicitly make it clear that we're not passing the error to the form though (because I think it's weird that we're not, and that the banners should actually be form field validation errors instead, but that's an argument I lost).

Can drop it if you think it's fine implicitly not passing the error through though.

Reply:
That makes sense - let's leave it as it is.